source/2019-11-01-ember-3-14-released.md

```diff
@@ -0,0 +1,184 @@
+```
+
+Specify the Octane edition in `.ember-cli.js` by adding the following lines to the beginning of the file:
+
+```js
+// .ember-cli.js
+const { setEdition } = require("@ember/edition-utils");
+
+setEdition("octane");
+```
+
+If you have an `.ember-cli` file instead of an `.ember-cli.js` file, you can convert it by renaming it to `.ember-cli.js`, then taking the existing JSON object and assigning it to `module.exports`. For example, if you have an `.ember-cli` file with:
+
+```json
+// .ember-cli
+{
+  "disableAnalytics": false
+}
+```
+
+The equivalent `.ember-cli.js` file would be:
+
+```js
+// .ember-cli.js
+module.exports = {
+  disableAnalytics: false
+};
+```
+
+If you need more information on how to migrate away from these legacy features, check out the [Octane release plan](/2019/08/15/octane-release-plan.html) blog post.
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.14
+
+Ember.js 3.14 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is one (1) new feature, 0 (0) deprecations, and several bugfixes in this version.
+
+#### New Features (1)
+
+**`@model` in Route Templates (1 of 1)**
+
+Route templates can now access the route's model with `@model` in addition to `this.model`. By treating `@model` as an argument, route templates better parallel component templates. Furthermore, the concept of teaching a `this` context for templates can be delayed.
+
+These motivations as well as the detailed design of this feature are described in [RFC 523](https://github.com/emberjs/rfcs/blob/master/text/0523-model-argument-for-route-templates.md).
+
+A template such as the following can now be refactored from using `this.model`:
+
+```hbs
+{{!-- The model for this route is the current user --}}
+
+<div>
+  Hi <img src="{{this.model.profileImage}}" alt="{{this.model.name}}'s profile picture"> {{this.model.name}},
+  this is a valid Ember template!
+</div>
+
+{{#if this.model.isAdmin}}
+  <div>Remember, with great power comes great responsibility!</div>
+{{/if}}
+```
+
+To use `@model`:
+
+```hbs
+{{!-- The model for this route is the current user --}}
+
+<div>
+  Hi <img src="{{@model.profileImage}}" alt="{{@model.name}}'s profile picture"> {{@model.name}},
+  this is a valid Ember template!
+</div>
+
+{{#if @model.isAdmin}}
+  <div>Remember, with great power comes great responsibility!</div>
+{{/if}}
+```
+
+The guides will soon be updated to teach the `@model` form first-and-foremost.
+
+#### Deprecations (0)
+
+There are no new deprecations in Ember.js 3.14.
+
+For more details on changes in Ember.js 3.14, please review the [Ember.js 3.14.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.14.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.14
```

Comment:
@igorT @dgeb no changes at all to Ember Data?

Reply:
I think there will be. I'll add the ember data + cli changes once they've been released. Currently, these sections are just empty boilerplate.

Reply:
@dgeb Any updates? Or do you need any support?

Reply:
> Any updates? Or do you need any support?

@kennethlarsen ember cli was released today and I just finished updating this post with both data + cli changes. A review would be much appreciated!